stories/forms/utils.ts

```diff
@@ -0,0 +1,200 @@
+export const countries = [
```

Comment:
nit: might name this file `constants.ts` and capitalcase `COUNTRIES`

Reply:
its just for our shared stories - i can, but we could also easily rm and use another example